packages/djangoflow_app/lib/src/presentation/listeners/app_cubit_listener.dart

```diff
@@ -8,6 +8,7 @@ class AppCubitListener extends BlocListener<AppCubit, AppState> {
     required super.listener,
     BlocListenerCondition<AppState>? listenWhen,
   }) : super(
+          bloc: AppCubit.instance,
```

Comment:
Same in many places you have added value for `bloc`. I think this is not needed.